ssl/ssl_local.h

```diff
@@ -1258,7 +1258,7 @@ struct ssl_connection_st {
     SSL_EARLY_DATA_STATE early_data_state;
     BUF_MEM *init_buf;          /* buffer used during init */
     void *init_msg;             /* pointer to handshake message body, set by
-                                 * ssl3_get_message() */
+                                 * tls_get_message_body() */
```

Comment:
This is also untrue - the tls_get_message_body uses the pointer, it does not set it.

Reply:
the correct function to mention here would be tls_get_message_header()

Reply:
Okay.

Reply:
@t8m I am also confused whether this type of change will come under the CLA: trivial? I will be glad if you could help me with this. Also thank you for the above reviews.

Reply:
So far I would call this trivial

Reply:
Okay.